### PR TITLE
Crableg Sprite Fix

### LINKED ISSUE #17735: [Bug]: Crab Legs Are Invisible
### Checks

- [X] I have searched the bug with a few keywords, and I confirm this bug was not yet reported.

### Location

Horizon

### Describe the issue

Crab legs are so rare, they're now invisable sprites. 
image
image

### How to reproduce

1) Order le crab
2) Have invisable crab legs
3) profit??

### Round ID

All of them

### Staff ckey

NobleRow

### PR DESCRIPTION
**Pull request by Acetrea**
Fixes #17735

### CODE REVIEW

**Comment by Acetrea** [comment]
!bugfix

**Comment by Acetrea** [comment]
!review

**Review by alsoandanswer — APPROVED** [review]

**Review by DreamySkrell — APPROVED** [review]

**Review by FluffyGhoster — APPROVED** [review]

File: html/changelogs/Acetrea-Crableg.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - bugfix: "Fixed Crablegs Sprite Not Showing"
```
Comment (SleepyGemmy) [inline html/changelogs/Acetrea-Crableg.yml:41]:
```suggestion
  - bugfix: "Fixed crablegs sprite not showing."
```

**Review by Arrow768 — APPROVED** [review]

**Comment by FluffyGhoster** [comment]
Merge master and that should solve the CI error

**Pull request merged**